integration-tests/custom-key/src/main/java/CustomKey.java

```diff
@@ -0,0 +1,9 @@
+public class CustomKey {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/sgoertzen/sonar-break-maven-plugin/file/2300516132/issues/source?bid=3070326&fileBranchId=3194199#l1)